tests/opencascade/nurbs_patches_manifold_00.cc

```diff
@@ -0,0 +1,81 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2014 - 2015 by the deal.II authors
+//
+//    This file is subject to LGPL and may not be distributed
+//    without copyright and license information. Please refer
+//    to the file deal.II/doc/license.html for the  text  and
+//    further information on this license.
+//
+//-----------------------------------------------------------
+
+// Read goteborg.iges and dump its topological structure to the
+// logfile.
+
+#include "../tests.h"
+#include <fstream>
+#include <deal.II/base/logstream.h>
+
+#include <deal.II/opencascade/boundary_lib.h>
+#include <deal.II/opencascade/utilities.h>
+#include <TopTools.hxx>
+#include <TopoDS_Shape.hxx>
+#include <Standard_Stream.hxx>
+
+using namespace OpenCASCADE;
+
+int main ()
+{
+  std::ofstream logfile("output");
+  deallog.attach(logfile);
+
+  std::vector<TopoDS_Face> faces;
+  std::vector<TopoDS_Edge> edges;
+  std::vector<TopoDS_Vertex> vertices;
+
+  TopoDS_Shape sh = read_STEP(SOURCE_DIR "/step_files/goteborg.step");
+  extract_geometrical_shapes(sh, faces, edges, vertices);
+
+  dealii::OpenCASCADE::NURBSPatchManifold<2,3> manifold(faces[0]);
+
+  auto bounds = manifold.get_uv_bounds();
+  auto u_min = std::get<0>(bounds);
```

Comment:
Can you replace these occurrences of `auto` by their proper type? We don't yet require C++11.